src/main/scala/scala/tools/partest/nest/Runner.scala

```diff
@@ -765,7 +765,7 @@ class SuiteRunner(
 
   def banner = {
     val baseDir = fileManager.compilerUnderTest.parent.toString
-    def relativize(path: String) = path.replace(baseDir, "$baseDir").replace(PathSettings.srcDir.toString, "$sourceDir")
+    def relativize(path: String) = path.replace(baseDir, s"$$baseDir").replace(PathSettings.srcDir.toString, "$sourceDir")
```

Comment:
I guess it's time to apply my `f"the familiar interpolator"` sticker to the lid of my lap top.